server/portal/apps/datafiles/views.py

```diff
@@ -236,44 +237,46 @@ def put(self, request, filetype):
 @method_decorator(login_required, name='dispatch')
 class LinkView(BaseApiView):
     def create_postit(self, request, scheme, system, path):
+
         client = request.user.tapis_oauth.client
-        body = {
-            "url": "{tenant}/files/v2/media/system/{system}/{path}".format(
-                tenant=settings.AGAVE_TENANT_BASEURL,
-                system=system,
-                path=path
-            ),
-            "unlimited": True
-        }
-        response = client.postits.create(body=body)
-        postit = response['_links']['self']['href']
+
+        # Create postit for unlimited use with a valid period of 1 year
+        postit = client.files.createPostIt(systemId=system, path=path, allowedUses=-1, validSeconds=31536000)
```

Comment:
Getting a permissions error. Is there a particular permission a user needs on a system to create postits?

> ForbiddenError: message: FILES_NOT_AUTHORIZED. Tenant: portals OboUserId: sal SystemId: frontera Path: home1/05089/sal/get_port.py Permission: READ